lib/filter/filter-expr-grammar.ym

```diff
@@ -103,12 +105,11 @@ filter_simple_expr
 	| KW_NETMASK '(' string ')'    { $$ = filter_netmask_new($3); free($3); }
   | KW_NETMASK6 '(' string ')'   {
   #if ENABLE_IPV6
-                                    $$ = filter_netmask6_new($3);
-  #else
-                                    YYERROR;
+    $$ = filter_netmask6_new($3);
```

Comment:
Error parsing filter expression, filter plugin netmask6 not found OR you may not used double quotes in your filter expression in....